src/main/java/org/broadinstitute/hellbender/tools/spark/sv/GATKSVVCFHeaderLines.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFHeaderLineCount;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GATKSVVCFHeaderLines {
+
+    public static Map<String, VCFHeaderLine> vcfHeaderLines = new HashMap<>();
+
+    // VCF standard SV header lines
+    // todo: add these and the other standard SV info fields from the VCF spec to htsjdk VCFStandardHeaderLines
+    public static final String SVTYPE = "SVTYPE";
+    public static final String SVLEN = "SVLEN";
+
+    // GATK-SV specific header lines
+    public static final String TOTAL_MAPPINGS = "TOTAL_MAPPINGS";
```

Comment:
While looking at the example VCF, I realized that this TOTAL_MAPPINGS field might be redundant, because this integer can always be inferred from other annotations.
I would guess that this is put here for convenience and may not lead to much spatial overhead.
Is this the case?